spec/system/product_drive_system_spec.rb

```diff
@@ -130,4 +130,34 @@
       expect(page).to have_content 'Endless drive'
     end
   end
+
+  context "when deleting a Product drive" do
```

Comment:
Can we move these tests to request tests? There's no reason to load a whole browser to test the HTML output by the controller.

Reply:
Got it! Thanks for the heads-up